PythonForDelphi/Components/Sources/Core/PythonGUIInputOutput.pas

```diff
@@ -50,7 +50,7 @@ interface
   Windows, Messages,
 {$ENDIF}
   SysUtils, Classes, Graphics, Controls, Forms, Dialogs,
-  StdCtrls, PythonEngine;
+  StdCtrls, PythonEngine, LCLType;

```

Comment:
This breaks Delphi compatibility.

Reply:
Ok. You could remove it now. I suppose that in this phase it is not necessary.